File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // pyramid rows
    for (int i = 0; i < height; i++)
    {
        // left-align space
        for (int j = height; j > i + 1; j--)
        {
            printf(" ");
        }

        // left-align pyramid column
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}

File: week6/sentimental-mario-less/mario.py
import cs50


def main():
    height = get_height()
    # Pyramid Rows
    for i in range(height):
        # Left-align Spaces
        print(" " * (height - (i + 1)), end="")
        # Pyramid Columns
        for k in range(i + 1):
            print("#", end="")
        print()


# Get Height From User
def get_height():
    while True:
        n = cs50.get_int("Height: ")
        if n > 0 and n < 9:
            return n


main()